INSTALL.md

```diff
@@ -1,40 +1,27 @@
 ## Installation Steps
 
-Helm charts are provided inside https://github.com/eclipse-tractusx/sd-factory
-
-1.) Using helm commands: <br />
+A helm chart is provided inside the [charts](https://github.com/eclipse-tractusx/sd-factory/tree/main/charts/sdfactory) directory
 
 How to install application using helm:  <br />
-    helm install ReleaseName ChartName
+
+              helm install [ReleaseName] [ChartName]
+
+1.) Installation from released chart: <br />
     
-    a.) Add helm repository in tractusx:
-           helm repo add sd-factory https://eclipse-tractusx.github.io/charts/dev
+    a.) Add helm repository for tractusx-dev:
+           helm repo add tractusx-dev https://eclipse-tractusx.github.io/charts/dev
     b.) To search the specific repo in helm repositories 
-           helm search repo sd-factory/sdfactory
-    c.) To install using helm command:
-           helm install sdf sd-factory/sdfactory
+           helm search repo tractusx-dev/sdfactory
+    c.) To set your own configuration and secret values, install the helm chart with your own values file in kubernetes cluster  
+           helm install -f your-values.yaml [ReleaseName] tractusx-dev/sdfactory -n [NameSpace]
+    d.) These secrets should be defined in Hashicorp vault to keep them secure.   
```

Comment:
I find a separate point for it confusing as this statement is just a clarification to 1.c, I'd just put it after 1.c.
Also, [this](https://github.com/eclipse-tractusx/sd-factory/pull/104/files#diff-09b140a43ebfdd8dbec31ce72cafffd15164d2860fd390692a030bcb932b54a0L23) list could be helpful, please re-add it

Reply:
This has been taken care